infra/main.parameters.json

```diff
@@ -43,6 +43,12 @@
     },
     "storageResourceGroupName": {
       "value": "${AZURE_STORAGE_RESOURCE_GROUP}"
+    },
+    "chatGptDeploymentName": {
+      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT}"
+    },
+    "gptDeploymentName": {
+      "value": "${AZURE_OPENAI_GPT_DEPLOYMENT}"
```

Comment:
Just wondering if you tried:

"value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT=chat}"

"value": "${AZURE_OPENAI_GPT_DEPLOYMENT=davinci}"

 
I thought I saw that as a way to specify defaults but haven't tested.
Assuming that doesnt work, your PR is the other way I would have tried.

Reply:
We should try this in a follow up and see if it works!